pages/people.md

```diff
@@ -18,19 +18,21 @@ permalink: "/people/"
 
 ## Developers ##
 
-- Yuichi Ike - Fujitsu Laboratories
-- [Th&eacute;o Lacombe][11]{:target="_blank"} - Postdoc at INRIA Saclay - Ile de France
-- [Siddharth Pritam][12]{:target="_blank"} - Postdoc at INRIA Sophia Antipolis - M&eacute;diterran&eacute;e
-- Masatoshi Takenouchi - Fujitsu Laboratories
+- [Th&eacute;o Lacombe][11]{:target="_blank"} - Assistant professor, LIGM, Université Gustave Eiffel
```

Comment:
Should we keep a mention of the fact that he already contributed as phd/postdoc?

Reply:
I added some details on ec538f8